### DISCUSSION TITLE
Local "Yarn build" working but GitHub Action "Test-deploy" shows - TypeError: Invalid URL

### DISCUSSION THREAD

**Discussion by MatthewJeffson** [facebook/docusaurus#9178] [Q&A] [answered]
Our website is working fine, until I merged a md file file when PR shows:  TypeError: Invalid URL

But the deploy and the website build are both fine, So I didn't think too much.

However, after that every PR and page deploy are all showing: TypeError: Invalid URL
<img width="640" alt="企业微信截图_71204b79-3792-4f83-85eb-10fae482c667" src="https://github.com/facebook/docusaurus/assets/104187183/63a3a2a2-19e0-4705-bbc3-73125bfcda3e">

The interesting part is that my local "yarn build" shows it all fine....

Now if I want to change my website, I only use the "build" file that I "yarn build" locally and push directly to the "gh-pages"...

Is it possible that I can locate where I have this "Unvalid URL"? Because I have reset everything back to the time that I didn't merge that file. And it keeps shows: TypeError: Invalid URL

Really confused, Hope someone can help me.

Thank you in advanced!!

**Comment by MatthewJeffson** [comment]
This is our code if any helps:https://github.com/Seeed-Studio/wiki-documents  Really need to solve this quick. Thank you very much🙏

**Reply by slorber** [reply]
I have no idea what happens, unfortunately. 
If you reverted everything maybe it's a temporary GitHub CI problem?
Maybe their action setup was upgraded or something? I can't tell. 

Try really reverting it or re-running an old workflow to see if it still works (like this workflow https://github.com/Seeed-Studio/wiki-documents/actions/runs/5655454595/job/15320607573)

**Comment by MatthewJeffson** [comment, marked as answer]
I found it!!!!!! causing two wholes days, day and night!

<img width="685" alt="image" src="https://github.com/facebook/docusaurus/assets/104187183/a30492fc-d656-4a13-a2fb-ed623d1cc471">

This is the reason causing my problem. That is the reason GitHub deploy function always shown url error and couldn't tell me where.

However, someone knew the principle of it?

**Reply by slorber** [reply]
I don't know what happens here, but noted to investigate later when I have time
